test/sqllogictest/audit_log.slt

```diff
@@ -107,5 +110,11 @@ SELECT id, event_type, object_type, details, user FROM mz_audit_events ORDER BY
 28  create  source  {"database":"materialize","id":"u7","item":"s","schema":"public","size":"1"}  materialize
 29  alter  source  {"database":"materialize","id":"u7","item":"s","new_size":"2","old_size":"1","schema":"public"}  materialize
 30  drop  source  {"database":"materialize","id":"u7","item":"s","schema":"public"}  materialize
-31  drop  cluster-replica  {"cluster_id":"u2","cluster_name":"foo","replica_name":"r"}  materialize
-32  drop  cluster  {"id":"u2","name":"foo"}  materialize
+31  create  source  {"database":"materialize","id":"u8","item":"organizations","schema":"public","size":null}  materialize
```

Comment:
Is this the desired output for subsources and billing? `null` size == unbilled? Has the cloud billing query been updated to account for this?

Reply:
Yes—it will be!

Reply:
cc @necaris

Reply:
Will `IS NOT NULL` behave correctly with a JSON `null`? https://github.com/MaterializeInc/cloud/pull/4282/commits/b7460b516c6cd62fb125c983dbe52de01efd8d66

Reply:
```
materialize=> create table t (j json);
CREATE TABLE
Time: 1093.320 ms (00:01.093)
materialize=> insert into t values (null), ('null');
INSERT 0 2
Time: 87.338 ms
materialize=> select j, j is null from t;
  j   | ?column? 
------+----------
      | t
 null | f
(2 rows)
materialize=> insert into t values ('"null"');
INSERT 0 1
Time: 160.212 ms
materialize=> select j, j is null, j = 'null' from t;
   j    | ?column? | ?column? 
--------+----------+----------
        | t        | 
 null   | f        | t
 "null" | f        | f
```

So `IS NULL` returns false on json null. You need to compare to `'null'::json`.

Reply:
https://github.com/MaterializeInc/cloud/pull/4282/commits/52abcd228edf5c0f988dd6c22b720eee6181c121 looks like it should do it, thanks @mjibson !